urls.env

```diff
@@ -1,7 +1,7 @@
 METADATA_URL=http://metadata/dev.xml
-POLICY_URL=http://policy
-CONFIG_URL=http://config
-SAML_PROXY_URL=http://saml-proxy
+POLICY_URL=http://policy:50110
+CONFIG_URL=http://config:50240
+SAML_PROXY_URL=http://saml-proxy:50220
```

Comment:
We have ports for some of these, but not others - but also we have a config/ports file. Let's talk about how it all fits together.

Reply:
I'm wondering a similar thing - what was the resolution?

Reply:
So these are fully qualified URLs which are used in the config.  The ports in the ports.env file tell the services generally what ports to run on and are sometimes used to configure the opening of ports in the docker-compose.yml